src/main/java/com/vejo/em/casa/be/rest/EventController.java

```diff
@@ -16,8 +15,8 @@
     EventService eventService;
 
     @GetMapping(value="/getAllEvents/{page}/{size}")
```

Comment:
path should be only /getAllEvents

page and size pagination should also be query parameters (@RequestParam). "page" and "size" should be mandatory